src/preview/send.ts

```diff
@@ -0,0 +1,26 @@
+import { send as sendMail } from '../helpers/mail';
+import templates from '../templates';
+import constants from '../helpers/constants.json';
+import { rpcSuccess, rpcError } from '../helpers/utils';
+
+export default async function send(req, res) {
+  const params = {
+    to: constants.example.to,
```

Comment:
Can you make `to` and `address` as url query param instead taking constants?

Reply:
We can, but this will be dangerous, as the endpoint is not locked behind authentication

Reply:
Right then maybe you can add a pass, could do with query params like ?key=blabla and check `if sha256(key) === 'ccadd99b16c...'`

Reply:
The reason is that we can hardcode the string encrypted with sha256 without exposing the secret